modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,54 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run many commands in parallel on a large number of clients.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting features to control the load in your environment.
+
+These features are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+
+
+== Batch Size
+
+Actions are processed in batches.
+This parameter sets the maximum number of clients that can execute a single action in each batch.
+
+Choosing a lower number can prevent system thrashing, but might reduce overall performance as parallelization is reduced.
+
+Adjust the [systemitem]``java.salt_batch_size`` parameter.
+Defaults to 200.
```

Comment:
@chiaradiamarcelo are we set on 200? Maybe we want to follow the traditional client default and change that to 100? I can't recall the final decision on this.

Reply:
As for now, the default parameter is 200, yes

Reply:
Edit, the default value at the moment is 100, same as the trad clients